### ISSUE TITLE
Select multiples strokes issue vector level with pen tablet

### ISSUE THREAD

**Issue by openanim** [opentoonz/opentoonz#3814]
**Describe the bug**
It seems there is an issue using the pen and select multiples strokes.  Most of the time it fails

**To Reproduce**
List steps to reproduce the behavior.
1.create a vector level
2. draw severals strokes
3.use selection tool and select  multiple strokes using shift key

**Screenshots**
![select multiple strokes issue](https://user-images.githubusercontent.com/29790512/112237154-5e22c800-8c42-11eb-9207-c49c1f1c993d.gif).

W 10 cintiq companion 2 , OT1.5 RC

**Comment by beeheemooth** [comment]
Looking at the nature of your two issues, I can assume that there are actually more of them (for example, highlighting a number with a stylus in the input window). I also had a similar problem with Wacom: when I tried to draw a geometric shape "Line" the program crashed when I made only one click. I solved the problem by reinstalling the driver.

**Comment by openanim** [comment]
Yes, I also think that. I also suspect the following two issues as possibly related with #3817.

In the 1st case I get a crash while creating a raster toonz level (maybe similar to what you had).

I cannot reproduce it during a session but it seems reproducible as soon as OT is opened with the following steps :

1. create a vector level

2. draw a few lines with the brush or the geometry tools

3. create a toonz raster level

![toonz raster level crash](https://user-images.githubusercontent.com/29790512/112329705-b6e07800-8cb7-11eb-88d1-3ea5947972c4.gif)

The crash occurs for me that with my stylus, with the mouse it seems to work normally

In the 2nd case, it became difficult to activate the level strip onion skin from the two semi circles.

1. from a level, create several frames

2. try to activate the onion skin semi circles.

![enable level trip onion skin_issue](https://user-images.githubusercontent.com/29790512/112329808-cc55a200-8cb7-11eb-84a4-707ecf7cf42d.gif)

I haven't tried updating my wacom driver yet, because later versions have a problem with opentoonz (I reported the problem on wacom but haven't since checked to see if this has been fixed).
